migration-engine/migration-engine-tests/tests/migrations/advisory_locking.rs

```diff
@@ -33,15 +33,15 @@ fn advisory_locking_works(mut api: TestApi) {
             // We move the engines into the async block so they get dropped when they
             // are done with the request, releasing the lock as a consequence.
             async move {
-                second_me
+                first_me
                     .apply_migrations(&migrations_directory)
                     .send()
                     .await
                     .unwrap()
                     .into_output()
             },
             async move {
-                first_me
+                second_me
```

Comment:
Why the reorder here?

Reply:
revert it: https://github.com/prisma/prisma-engines/pull/3308/commits/8177d21bfa7a5699437ae5d4915d946a0ed81f3b